versioned_docs/version-22.04/alerts-notifications/custom-views.md

```diff
@@ -70,7 +70,7 @@ To share a view with "locked"/"unlocked users" :
 
 ## Displaying a custom view users have shared with you
 
-1. On the **Home > Custom Views** page, click the **Show/Hide edit mode** icon in the top right corner of the page: <img src={require('../assets/alerts/edit_mode.png').default} style={{width:'14px'}} />
+1. On the **Home > Custom Views** page, click the **Show/Hide edit mode** icon in the top right corner of the page: ![image](../assets/alerts/edit_mode.png)
```

Comment:
fr section is not exactly same image inclusion : `![image](../assets/alerts/edit_mode.png#thumbnail1)`